simpleLooping/createBinaryStringList.py

```diff
@@ -7,3 +7,19 @@
 #   createBinaryStringList(4) # return ["0", "1", "10", "11"]
 #   createBinaryStringList(6) # return ["0", "1", "10", "11", "100", "101"]
 #   createBinaryStringList(8) # return ["0", "1", "10", "11", "100", "101", "110", "111"]
+from typing import List
+
+def createBinaryStringList(N: int) -> List[str]:
+    if N <= 0:
+        return []
+
+    binaryStringList = ["0"]
+    for num in range(1, N):
+        binaryString = ""
+        while num > 0:
+            binaryString = str(1 if num % 2 else 0) + binaryString
+            num >> 1
```

Comment:
Did you rerun unittest for this one? If you did, you will see that if num = 5 then run "num >> 1" will not change num to 2.

Why? because you did the operation but you did not assign the result back to num. 

In other words, does 2 code block below equal?

    num = 5
    num = num >> 1
    print(num)

vs

    num = 5
    num >> 1
    print(num)